api/SVGSetElement.json

```diff
@@ -6,7 +6,7 @@
         "spec_url": "https://svgwg.org/specs/animations/#InterfaceSVGSetElement",
         "support": {
           "chrome": {
-            "version_added": "1"
+            "version_added": "2"
```

Comment:
svg.elements.set still says "1". It's possible that the `<set>` markup was supported before the `SVGSetElement` API, but I've dug into WebKit history and it seems like both the markup and the API were supported before Chrome 1: https://github.com/WebKit/WebKit/commit/f21933ab8d8e07f1ca52d0962f9cea643cae6842

This makes me suspicious of all of the 1&rarr;2 changes for SVG. For interfaces that don't have any members it's kind of meaningless to say it's not supported, since there's no constructor and the interface name is only visible if one really looks closely.

Reply:
I've cherry-picked these changes to #19152 to help get this one landed easier!